src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -40,36 +40,61 @@ export const applicantListSchema = (requireds, propertyList) => {
   };
 };
 
-/**
- * Gets the appropriate form key needed in the signature box so we can
- * corroborate who is signing the form.
- *
- * @param {object} formData All data currently in the form
- * @returns
- */
-export function getNameKeyForSignature(formData) {
-  let nameKey;
-  if (formData.certifierRole === 'sponsor') {
-    nameKey = 'veteransFullName';
-  } else if (formData.certifierRole === 'applicant') {
-    nameKey = 'applicants[0].applicantName';
-  } else {
-    nameKey = 'certifierName';
-  }
-
-  return nameKey;
-}
-
```

Comment:
There is now no question about who is signing the form, so we can nix this logic and always require the certifier's full name instead.